rest/route/host_spawn.go

```diff
@@ -156,6 +158,19 @@ func (h *hostModifyHandler) Run(ctx context.Context) gimlet.Responder {
 		return gimlet.MakeJSONErrorResponder(errors.Wrapf(err, "Error creating spawnhost modify job"))
 	}
 
+	if h.options.SubscriptionType != "" {
+		var subscriber event.Subscriber
+		if h.options.SubscriptionType == event.SlackSubscriberType {
+			subscriber = event.NewSlackSubscriber(fmt.Sprintf("@%s", user.Settings.SlackUsername))
+		} else if h.options.SubscriptionType == event.EmailSubscriberType {
+			subscriber = event.NewEmailSubscriber(user.Email())
+		}
+		resultSubscription := event.NewSpawnHostStateChangeOutcomeByHost(h.hostID, subscriber)
```

Comment:
can we error if the subscription type is wrong? Otherwise it looks like we'll create a subscription with a nil subscriber. Also the user might want to know if they typoed or something.

Reply:
should be fixed now: the user will get back an error response from the server.